spec/offline_sort/offline_sort_spec.rb

```diff
@@ -1,113 +1,138 @@
 require 'spec_helper'
 
 describe OfflineSort::Sorter do
+  describe "#sort" do
+    subject { offline_sorter_instance.sort.to_a }
```

Comment:
With the change to return a `Merger`, I think that we'll basically want to call `to_a` on the return of `sort` multiple times to confirm that we're not actually running things multiple times (as opposed to calling `.sort` on `offline_sorter_instance` multiple times as I think the current set up does). Is that lining up with your understanding of these changes too or am I missing something there?

Reply:
This is only relevant to the tests where we're explicitly making assertions on calling this multiple times, right? If so, made that change below, let me know what you think!

Reply:
Discussed live: going to only replace this subject in the multiple sort test case, as this makes the comparisons easier to write in the other specs